02-microarray/pathway-analysis_microarray_01_ora.Rmd

```diff
@@ -170,11 +175,15 @@ library(magrittr)
 
 ## Import data
 
-We will read in the differential expression results we will download from online.
+For ORA, we only need a list of gene IDs as our input, so this example can work for any situations where you have gene list and want to know more about what biological pathways it shares genes with.
+
+For this example, we will read in the differential expression results we will download from online.
```

Comment:
```suggestion
For this example, we will read in results from a differential expression analysis that we have already performed.
Rather than reading from a local file, we will download the results table directly from a URL.
```